src/james.data.clss.pas

```diff
@@ -207,13 +207,16 @@ function TDataStream.Save(const FileName: string): IDataStream;
 end;
 
 function TDataStream.AsString: string;
+var
+  Buf: AnsiString;
 begin
   with FStream do
   begin
-    SetLength(Result, Size);
+    SetLength(Buf, Size);
     Position := 0;
-    ReadBuffer(Pointer(Result)^, Size);
+    ReadBuffer(Pointer(Buf)^, Size);
```

Comment:
I don't see why not. 
However, TStringStream interface in Delphi differs from the one in FPC, so I would recommend creating a class to replace it in each platform.
What do you think?